examples/addon_example/lib/manual.widgetbook.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:addon_example/app.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+
+void main(List<String> args) {
+  runApp(const HotReload());
+}
+
+class HotReload extends StatelessWidget {
+  const HotReload({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Widgetbook.material(
+      directories: [
+        WidgetbookComponent(
+          name: 'Comp',
+          useCases: [
+            WidgetbookUseCase(
+              name: 'Use-case',
+              builder: (context) {
+                return Text('The sound is: ${context.sound}');
+              },
+            )
+          ],
+        ),
+      ],
+      addons: [
+        MaterialThemeAddon(
+          setting: ThemeSetting.firstAsSelected(
+            themes: [WidgetbookTheme(name: 'Dark', data: ThemeData.dark())],
+          ),
+        ),
+      ],
+      addons2: [
+        SoundAddon(),
+      ],
```

Comment:
🚨 I added another property for the alternative and refactored API so I don't need to migrate all existing addons. This will not be in the final product.